### ISSUE TITLE
VariableOverflow false positive

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#999]
On behalf of RITS

Rule VariableOverflow is triggered with message `Conversion from DECIMAL to INT can overflow if source expression is larger than 2^32` on this statement:
```
define input parameter xxx as decimal.
assign frame f1:height-chars = xxx.
```

Not reproduced internally, but visible on live instance.

**Comment by gquerret** [comment]
Duplicate of #1028
